Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessClusterManager.java

```diff
@@ -72,7 +72,7 @@ public static AzureSparkServerlessClusterManager getInstance() {
     private static final String REST_SEGMENT_ADL_ACCOUNT = "providers/Microsoft.DataLakeAnalytics/accounts";
 
     // FIXME!!!
-    private static final String ACCOUNT_FILTER = CommonSettings.getAdEnvironment().endpoints().getOrDefault("dataLakeSparkAccountFilter", "");
+    private static final String ACCOUNT_FILTER = CommonSettings.getAdEnvironment().endpoints().getOrDefault("dataLakeSparkAccountFilter", "endswith(name, '-c09')");
```

Comment:
>"endswith(name, '-c09')" [](start = 138, length = 24)

The matching should be something like ".*-c\d{2}"

Reply:
The cluster name can be vc1-c09, vc2-c14

---
In reply to: [201975278](https://github.com/Microsoft/azure-tools-for-java/pull/1763#discussion_r201975278) [](ancestors = 201975278)